std_msgs/Header header
sensor_msgs/PointCloud2 cloudMap
sensor_msgs/PointCloud2 cloudObstacles
map_merging/Index[] clusterList

